CMakeLists.txt

```diff
@@ -47,6 +47,10 @@ if(NOT CMAKE_CXX_FLAGS_RELEASE MATCHES "-O")
   set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3")
 endif()
 
+if(APPLE)
+  set(CMAKE_PREFIX_PATH /opt/pkg)
```

Comment:
Why is this needed? Is your cmake in `/opt/pkg/bin`? If so, cmake should find files under `/opt/pkg`.